### PR TITLE
[Ready] Code Bounty: Fake Virus Event

### PR DESCRIPTION
**Pull request by PKPenguin321**
**What is it**
A new event that will target 1-3 people randomly. The fluff text/emotes from multiple viruses (such as coughing, sneezing, dizziness, confusion, vomiting (just the fluff text no actual vomit), etc) will be thrown at the victims as with a regular virus, but they will not actually be suffering from a virus.

The way it works is that it gives a 3 minute invisible status effect that will throw random spooky messages with increasing spookiness over time (like a real virus would). After the 3 minutes is up, the status effect vanishes and it's like you were never even afflicted. Also, after the 1-3 random people are given this status effect, 1-5 more are made to cough/sneeze/etc exactly once with a randomized brief delay.

**Purpose**
This should hopefully make people stop and think before immediately going to lynch the virologist when they see a "You feel lightheaded!" message. Adds paranoia.

:cl:
add: A new Fake Virus event that makes you think you have a virus when you don't
/:cl:

### CODE REVIEW

File: code/datums/status_effects/debuffs.dm [hunk 1]
```diff
@@ -740,3 +740,46 @@
...
+
+/datum/status_effect/fake_virus
+	id = "fake_virus"
+	duration = 1800//3 minutes
+	status_type = STATUS_EFFECT_REPLACE
+	tick_interval = 1
+	alert_type = null
+	var/msg_stage = 0//so you dont get the most intense messages immediately
+
+/datum/status_effect/fake_virus/tick()
```
Comment (Qustinnus) [inline code/datums/status_effects/debuffs.dm]: wouldnt it be obvious from the amount of symptoms that this isnt a real virus?

Comment (PKPenguin321) [reply code/datums/status_effects/debuffs.dm]: That may be the case and I was going to address it but it's possible that pretty much this whole PR will have to be rewritten anyways should the bounty author want it done differently

**Comment by optimumtact** [comment]
This doesn't seem like it would be fun or interesting at all

**Comment by Qustinnus** [comment]
@optimumtact it prevents people from auto-guessing there's a virus if someone coughs. adding multiple causes for events prevents meta

**Comment by YakumoChen** [comment]
The original intention was for people to cough or something once or twice to instill paranoia, but I'm honestly not sure if this is better or worse as an event.

**Comment by Ratvar** [comment]
"This should hopefully make people stop and think before immediately going to lynch the virologist when they see a "You feel lightheaded!" message. "
... you sure everyday TG player won't just rush and try to kill virologist/chemist/whoever because they are totally infected, and then lie to admins they thought they were infected?

**Comment by PKPenguin321** [comment]
IC issue

**Comment by Ratvar** [comment]
But is it a *good* thing to include such issue?

**Comment by PKPenguin321** [comment]
Yes, the game is built from conflict

**Comment by ATH1909** [comment]
> Real talk though, if you're too hasty to even use a medical analyzer on yourself

Stealth 2+ viruses are a thing.

Even as a virologist main, I dunno if this PR is a good idea. Why are we punishing people for reacting appropriately to a potential deathvirus outbreak?

Better idea here: Instead of making your proposed status effect come from a random event, add a new type of hallucination effect (y'know, like fake speech, hallucinatory bubblegum, hallucinatory shocks, etc.) that applies your proposed status effect for a period of time.

**Comment by Qustinnus** [comment]
no theres too much hallucinations already so itll be super rare

**Comment by ATH1909** [comment]
> no theres too much hallucinations already so itll be super rare

IMO, there aren't ENOUGH hallucinations currently. You end up seeing repeats often after you play for a while.

Also, if the issue is with it being super rare, you could just... bump up the weighting of the fake disease hallucination. There's a reason why hearing voices is more common than getting the OH YEAH hallucination.

**Comment by PKPenguin321** [comment]
Latest commit [(as per the bounty author's request)](https://tgstation13.org/phpBB/viewtopic.php?f=5&p=504122#p504094):
>After the 1-3 random people are given this status effect, 1-5 more are made to cough/sneeze/etc exactly once with a randomized brief delay.

File: code/datums/status_effects/debuffs.dm (hunk 1 above)
Comment (PKPenguin321) [reply code/datums/status_effects/debuffs.dm]: After testing a bit more I've decided that it's actually close enough to resemble a real virus that this won't be a big issue

**Comment by ErwinSnowden** [comment]
give it gbs and rhumba beat fake symptoms to get players made at admins

**Comment by Dawson1917** [comment]
Anything that makes it harder to immediately metagame that there's a sentient disease / rogue viro whenever you sniffle is a good addition to the game

**Comment by Couls** [comment]
Why not add it to hallucinations instead?

**Comment by Tlaltecuhtli** [comment]
miasma already spawns a bunch of viruses

**Comment by PKPenguin321** [comment]
Can the DNM tag be removed please

**Comment by MimicFaux** [comment]
Time for some anecdotal evidence why this PR was a great idea.
Was afflicted by this a couple days ago, was one of the only security available to the crew. The symptoms put me on guard, and their increasing intensity got me worried but I was in mid pursuit of the saboteur. Caught them, and deputized the clown with my hat and pepperspray to deliver the prisoner while I hauled ass to medbay only to stir up a panic among the doctor staff about a stealthy virus.
10/10 classic ss13 gameplay.

**Pull request merged**